### ISSUE TITLE
Feature: Ajax Reload Highlighter

### ISSUE THREAD

**Issue by lmcdn** [urvanov-ru/crayon-syntax-highlighter#31]
Sorry here again.
How can I fire the Highlighter when using ajax to show a post page.

Does this in your code works? I also use ajax to "show more posts, but sometimes it won't work.

```
$(document).ready(function () {
        UrvanovSyntaxHighlighterSyntax.init();
    });
```

**Comment by urvanov-ru** [comment]
@lmcdn yes, this code is initialization for highlighter. I don't know, can it work with ajax or no. I never used it in my site.
